src/main/features/claim/views/eligibility/not-eligible.njk

```diff
@@ -0,0 +1,60 @@
+{% extends "layout.njk" %}
+
+{% set title = t('Not eligible | Money claim') %}
+
+{% macro newWindow() %}<span class="font-xsmall">{{ t('(opens in new window)') }}</span>{% endmacro %}
+
+{% block content %}
+
+  {% set n1FormUrl = 'https://formfinder.hmctsformfinder.justice.gov.uk/n1-eng.pdf' %}
+  {% set n510Url = 'http://formfinder.hmctsformfinder.justice.gov.uk/n510-eng.pdf' %}
+  {% set citizensAdviceUrl = 'https://www.citizensadvice.org.uk/about-us/contact-us/' %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">{{ t('You can’t use this service') }}</h1>
+
+      {% if reason === NotEligibleReason.CLAIM_VALUE_NOT_KNOWN %}
```

Comment:
Don't like the fact it is so easy to amend query string argument and change what screen renders. I would prefer to hide it out of the user hands. No big impact though.

Reply:
I can't see an issue with it, its not using the user entered data only specific values.
saves creating 7 more pages